### PR TITLE
Minor change, using analitycal solution for dynamic tests

### PR DESCRIPTION
**Pull request by loumalouomega**

### CODE REVIEW

File: applications/StructuralMechanicsApplication/tests/dynamic_test/dynamic_newmark_test_parameters.json [hunk 1]
```diff
@@ -49,36 +49,49 @@
...
-    "_json_output_process" : [
+    "_list_other_processes" :[],
+    "_print_output_process" : [
     {
         "python_module"   : "json_output_process",
         "kratos_module" : "KratosMultiphysics",
         "help"                  : "",
         "process_name"          : "JsonOutputProcess",
         "Parameters"            : {
             "output_variables" : ["DISPLACEMENT_X","VELOCITY_X","ACCELERATION_X"],
```
Comment (msandre) [inline applications/StructuralMechanicsApplication/tests/dynamic_test/dynamic_newmark_test_parameters.json]: Shouldn't the output file name be newmark not bossak?

File: applications/StructuralMechanicsApplication/tests/dynamic_test/dynamic_newmark_test_parameters.json [hunk 2]
```diff
@@ -49,36 +49,49 @@
...
     }
     ],
-    "json_check_process" : [
+    "_check_json_results_process" : [
     {
         "python_module"   : "from_json_check_result_process",
         "kratos_module" : "KratosMultiphysics",
         "help"                  : "",
         "process_name"          : "FromJsonCheckResultProcess",
         "Parameters"            : {
```
Comment (msandre) [inline applications/StructuralMechanicsApplication/tests/dynamic_test/dynamic_newmark_test_parameters.json]: also here.

**Comment by loumalouomega** [comment]
That part can be gone now

**Review by RiccardoRossi — COMMENTED** [review]
I have no comments on my side. I'll let Mike approve

**Comment by msandre** [comment]
@loumalouomega I don't understand what you mean. Can you please clarify?

**Comment by loumalouomega** [comment]
The check can be done analytically, then these lines can be gone. I have done that

**Review by msandre — APPROVED** [review]

**Pull request merged**